app/models/lunch.rb

```diff
@@ -85,18 +86,24 @@ def chalmrest
       allergenId = "id"
 
       restaurants.map do |restaurant|
-        data = JSON.parse(open(restaurant[:url]).read)
+        data = JSON.parse(open(url + restaurant[:id].to_s).read)
         if data.key?(items) && data[items] != nil
           meals = data[items].map do |category|
 
             category[recipes].map do |recipe|
               allergens = recipe[allergensKey]
-              summary = recipe[recipeNames].first[recipeName]
+              if locale == "sv"
+                displayName = category[name]
+                summary = recipe[recipeNames].first[recipeName]
+              else
+                displayName = category[nameEnglish]
+                summary = recipe[recipeNames][1][recipeName]
```

Comment:
`.second`? 😉

Reply:
Didn't know that was a thing ^^ Will fix! 😉

Reply:
It's where you find the links to the APIs for each restaurant.